sdk/healthdataaiservices/azure-health-deidentification/README.md

```diff
@@ -0,0 +1,163 @@
+# Azure Deidentification client library for Java
+
+Azure Deidentification client library for Java.
+
+This package contains Microsoft Azure Deidentification client library which is a managed service that enables users to tag, redact, or surrogate health data.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-health-deidentification;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-health-deidentification</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+## Key concepts
+### Operation Modes:
+
+- Tag: Will return a structure of offset and length with the PHI category of the related text spans.
+- Redact: Will return output text with placeholder stubbed text. ex. `[name]`
+- Surrogate: Will return output text with synthetic replacements.
+    - `My name is John Smith`
+    - `My name is Tom Jones`
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Azure Deidentification client use cases, including:
+
+- [Create a `DeidentificationClient`](#create-a-deidentificationclient)
+- [Calling deidentification endpoint](#calling-deidentification-endpoint)
+- [Creating deidentification Job](#creating-deidentification-job)
+- [Process deidentification Job](#process-deidentification-job)
+- [List deidentification Jobs](#list-deidentification-jobs)
+- [List completed files](#list-completed-files)
+
+### Create a `DeidentificationClient`
+
+Create a `DeidentificationClient` using the `DEID_SERVICE_ENDPOINT` environment variable.
+
+```java com.azure.health.deidentification.readme
+DeidentificationClientBuilder deidentificationClientbuilder = new DeidentificationClientBuilder()
+    .endpoint(Configuration.getGlobalConfiguration().get("DEID_SERVICE_ENDPOINT", "endpoint"))
+    .httpClient(HttpClient.createDefault())
+    .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC));
+
+DeidentificationClient deidentificationClient = deidentificationClientbuilder.buildClient();
+```
+
+### Calling `Deidentification` endpoint
+
+Calling the realtime endpoint with an input.
+
+```java com.azure.health.deidentification.sync.helloworld
+String inputText = "Hello, my name is John Smith.";
+
+DeidentificationContent content = new DeidentificationContent(inputText);
+content.setOperation(OperationType.SURROGATE);
+content.setDataType(DocumentDataType.PLAINTEXT);
```

Comment:
Since hello_world sample should be the smallest amount of code possible, we may want to remote these to setOperations since they are both optional now.